packages/playground-examples/copy/pt/JavaScript/Functions with JavaScript/Generic Functions.ts

```diff
@@ -0,0 +1,100 @@
+// Tipos genéricos provêm uma forma de usar Tipos como variáveis em outros
+// tipos. Meta.
+
+// Tentaremos manter este exemplo leve. Você pode fazer
+// muita coisa com tipos genéricos e é provável que veja algum código bem
+// complicado usando tipos genéricos em algum ponto - mas isso
+// não significa que tipos genéricos são complicados.
+
+// Vamos começar com um exemplo onde envolveremos um objeto de entrada
+// em um array. Apenas nos importaremos com uma só variável neste
+// caso: o tipo que foi passado como argumento:
+
+function envolverEmArray<Tipo>(entrada: Tipo): Tipo[] {
+  return [entrada];
+}
```

Comment:
@danilofuchs is there any standard approach on translating the code? In other places the code identifiers are kept as in the original language.

Reply:
Translating the code is great, as it reduces the barrier of entry. But, as it takes a lot of effort to translate, we are leaving it up to translators.

Reply:
__I__, generally ask to translate the code if it's small or simple enough, but when code is very complex or very long we can keep it that way